sr_moveit_hand_config/scripts/sr_moveit_hand_config/generate_hand_srdf.py

```diff
@@ -96,12 +84,33 @@ def __init__(self, urdf_str=None, load=True, save=True):
                 thumb = True
             if is_lite and key.endswith("WRJ2"):
                 is_lite = False
+        hand_name = side + "_hand"
+        
+        param = side+'_tip_sensors'
+        while not rospy.has_param(param):
+            rospy.sleep(0.5)
+            rospy.loginfo(f"waiting for {str(param)}")
```

Comment:
I don't think you need the `str()` if you are using f-strings. But maybe there's a reason here?

Reply:
Fixed